airflow/providers/hashicorp/_internal_client/vault_client.py

```diff
@@ -74,7 +74,7 @@ class _VaultClient(LoggingMixin):
     :param key_id: Key ID for Authentication (for ``aws_iam`` and ''azure`` auth_type).
     :param secret_id: Secret ID for Authentication (for ``approle``, ``aws_iam`` and ``azure`` auth_types).
     :param role_id: Role ID for Authentication (for ``approle``, ``aws_iam`` auth_types).
-    :param arn_role: AWS arn role (for ``aws_iam`` auth_type)
+    :param role_arn: AWS arn role (for ``aws_iam`` auth_type)
```

Comment:
Should we mark it as a breaking change?

Reply:
why breaking change? This is new param

Reply:
ah, you're right; I thought it already existed. Thanks for the clarification